### PR TITLE
add hpack

### PR DESCRIPTION
**Pull request by PrimozGodec**
required in https://github.com/conda-forge/staged-recipes/pull/4742

### CODE REVIEW

File: recipes/hpack/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - test
```
Comment (synapticarbors) [inline recipes/hpack/meta.yaml]: You should import `hpack` rather than `test`

File: recipes/hpack/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+about:
+  home: http://python-hyper.org/hpack/
+  license: MIT
```
Comment (synapticarbors) [inline recipes/hpack/meta.yaml]: add `license_file: LICENSE`

File: recipes/hpack/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  summary: 'HTTP/2 Header Encoding for Python'
+  dev_url: https://github.com/python-hyper/hpack
+
+extra:
+  recipe-maintainers:
+    - primozgodec
+    - astaric
```
Comment (synapticarbors) [inline recipes/hpack/meta.yaml]: @astaric -- are you ok being added as a co-maintainer of the `hpack` recipe on conda-forge?

https://conda-forge.org/docs/recipe.html#maintainer-role

File: recipes/hpack/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```
Comment (dougalsutherland) [inline recipes/hpack/meta.yaml]: This [should be `noarch: python`](https://conda-forge.org/docs/meta.html#building-noarch-packages), I think.

File: recipes/hpack/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - hpack
+
+about:
+  home: http://python-hyper.org/hpack/
+  license: MIT
```
Comment (dougalsutherland) [inline recipes/hpack/meta.yaml]: Not real important, but you could add `license_family: MIT` too.

**Comment by dougalsutherland** [comment]
Looks good to me now. Ready to merge as soon as we hear from @astaric; alternatively, you can drop him and merge now. (Adding him later is then easy.)

**Comment by jakirkham** [comment]
Had to restart Travis CI as it got really fouled up trying to checkout the repo for 8 mins. Nothing we can do about it, but would appreciate if people keep an eye out for this issue cropping up again. Saw a similar issue on Travis CI yesterday. If it keeps cropping up, we should follow-up with Travis CI.

Thanks all and happy new year! :)

**Pull request merged**